README.rst

```diff
@@ -17,11 +17,11 @@ This repository contains a course and some libraries that you can import into yo
      - Download
      - Source OLX
    * - *Open edX demo Course*
-     - ``course-v1:demo+Course+1``
+     - ``course-v1:Axim+DemoX+demo_course``
      - `<./dist/demo-course.tar.gz>`_
      - `<./demo-course>`_
    * - *Open edX Example Content Library*
-     - ``library-v1:demo+ContentLibrary``
+     - ``library-v1:Dummy+RESP_Q1``
```

Comment:
To me, "Demo+ContentLibrary" makes more sense than your suggestion, but you know libraries way better than me. Why "RESP_Q1"?

Reply:
AH I'm sorry, I didn't read carefully enough >_<

I agree that it would be preferable to use your suggestion here, and mine above, if that change can be made.